### PR TITLE
Team DelSquared Blogpost

### PR DESCRIPTION
**Pull request by arushi019**

### CODE REVIEW

File: blog/_posts/DelSquared_RGSoC_blogpost.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿**DelSquared: Rails Girls Summer of Code 2018 and Us**
```
Comment (v-thakkar) [inline blog/_posts/DelSquared_RGSoC_blogpost.md]: Hi @arushi019  ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.

File: blog/_posts/DelSquared_RGSoC_blogpost.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+We form a great, happy go lucky team and having daily conversations on our Telegram group relieves us of the tiredness after a long day’s work. We sometimes check our phones to see 50 odd messages of pure randomness on the group and it is somehow never annoying. 
+We worked together during the application process and all the revisions and the reviews were really smooth. Since the commencement of the internship, Brihi and Shravika have been talking to us about Tessel and the resources we should take a look at. They also talk highly of Kelsey, our mentor, who by the way is an absolute delight. We have had just one call with her till now and have another one scheduled for tomorrow. We are yet to talk to our supervisor, Bhumika, but we have a call scheduled for the end of this week. A horribly timed commitment has impeded our full-fledged start into the internship. However, we aim to compensate for the time lost over the subsequent weeks. Thanks, Vaishali, for granting us this extension :) Finally, a huge shout out to the entire RGSoC team for this wonderful initiative. We are really thrilled to be a part of this extraordinary community. Looking forward to a great summer. Cheers!
```
Comment (v-thakkar) [inline blog/_posts/DelSquared_RGSoC_blogpost.md]: Here, as we are going to publish your blog post in upcoming week(s), can you please also change the narrative here about the first call as by now I think it has already happened. Also, no need to specify about thanking me for the extension in the blog post. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by arushi019** [comment]
Hi @nerdyvaishali I have made the changes as per the review. Kindly check :)

**Review by v-thakkar — APPROVED** [review]

File: blog/_posts/DelSquared_RGSoC_blogpost.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿
+---
```
Comment (v-thakkar) [inline blog/_posts/DelSquared_RGSoC_blogpost.md]: Hi @arushi019 ! No need of the blank line at the start of an article and blank spaces at the start of parameters. It changes the format. Please test the blog post for markdown format. You can test it in some online tool also if you don't want to run the site locally.

File: blog/_posts/DelSquared_RGSoC_blogpost.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ layout: post
+ created_at: Fri Jul 10 2018
+ permalink: blog/DelSquared_RGSoC_blogpost
+ current: blog
+ author: Arushi
+ twitter: arushi1803
+ categories:
+ - student-posts
+ ---
+ **DelSquared: Rails Girls Summer of Code 2018 and Us**
```
Comment (v-thakkar) [inline blog/_posts/DelSquared_RGSoC_blogpost.md]: Please user markdown style headers and add blank lines in between each header and text. The comment applies to all titles and text in the whole file.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by arushi019** [comment]
Hi! 
I think this time it is okay and matches with the required format. @nerdyvaishali I am really sorry for troubling you again but please review :)

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
@arushi019 No problem. Looks good to me. Thanks!

**Pull request merged**